apps/web/utils/helpers/apiHelper.ts

```
@@ -9,4 +11,33 @@ const checkURL = (url: string): boolean => {
   return false;
 };
 
-export { checkURL };
+// eslint-disable-next-line no-undef
+function genApiKey(size = 20, format: BufferEncoding = 'base64') {
+  const buffer = randomBytes(size);
+  return buffer.toString(format);
+}
+
+async function genSecretHash(key: string) {
+  const hashStore = await hash(key);
+
+  return hashStore;
+}
+
+async function compareKeyAgainstHash(storedKey: string, suppliedKey: string) {
+  try {
+    if (
+      await verify(
+        '$argon2id$v=19$m=65536,t=3,p=4$2kasxHK6CP+NaXiw9P6J/A$GhgQWBtm5Whof9dI/ZVSfeWIj6dV+VdQ8EtmimOHxyw',
+        'VnIKCuUiGondxkjOsxg1ozTnnvM=',
+      )
```

whats this for?

---

ah, this was me just testing the verify method using hardcoded values. I called this method on a frontend page and that's where that nasty argon2 error came up thinking I messed up the depend again, but since it doesn't need to be ran on the frontend 🤦‍♂️ I shouldn't have even implemented it on a frontend page. (I'll remove the hard coded values).